Design.md

```diff
@@ -174,6 +174,14 @@ Originally discussed in [#12](https://github.com/coreos/fedora-coreos-tracker/is
  - For the short term, if we need to include an agent we will bake it into the image. We will not have any specific
    mechanism for including agents.
 
+### Azure:
+
+Originally discussed in [#65](https://github.com/coreos/fedora-coreos-tracker/issues/65).
+
+- We've identified one major gap with not shipping the [Microsoft Azure Linux Agent](https://github.com/Azure/WALinuxAgent): the machine will not check-in and will eventually be culled by Azure for being stuck in the creation process.
+- This gap will be covered by work done in [coreos-metadata](https://github.com/coreos/coreos-metadata/issues/120).
+- One additional gap which will __not__ be covered is a lack of ephemeral disk support. We plan to ship udev rules but will not have a service which formats the disk unless we receive feature requests in the future. This was discussed in [#97](https://github.com/coreos/fedora-coreos-tracker/issues/97).
```

Comment:
i don't know if there is any link we can provide to users who don't know what *ephemeral disk support* is. If we can that would be nice to have here. If not then it's fine too.

Reply:
Some quick googling led to https://blogs.msdn.microsoft.com/mast/2013/12/06/understanding-the-temporary-drive-on-windows-azure-virtual-machines/ but the concept of ephemeral storage is pretty consistent among cloud providers (for instance: AWS info here https://heitorlessa.com/working-with-amazon-aws-ec2-ephemeral-disks-fcefd46e4ec)

Reply:
right. I think users will know what ephemeral disks are, but they might not know what *ephemeral disk support* means. Never having used Azure much, and mostly working with AWS in the past, I didn't know what was meant by that phrase. I think we're good here for now though since later in the paragraph we mention formatting the disk(s).